staging/src/github.com/openshift/openshift-apiserver/pkg/project/auth/cache.go

```diff
@@ -195,6 +195,10 @@ type AuthorizationCache struct {
 
 	syncHandler func(request *reviewRequest, userSubjectRecordStore cache.Store, groupSubjectRecordStore cache.Store, reviewRecordStore cache.Store) error
 
+	informerCachesToSync     []cache.InformerSynced
+	informerCachesSynced     bool
+	informerCachesSyncedLock sync.RWMutex
```

Comment:
We the bool and lock simply because we do not die on sync failure?

Reply:
not sure what you mean?